TFT/src/User/API/interfaceCmd.c

```diff
@@ -724,6 +724,13 @@ void sendQueueCmd(void)
           uint8_t i = 0;
           if (cmd_seen('T')) i = cmd_value();
           if (cmd_seen('D')) setParameter(P_FILAMENT_SETTING, 1 + i, cmd_float());
+          if (infoMachineSettings.firmwareType == FW_SMOOTHIEWARE) 
```

Comment:
In all your code, for the inline comments, use 2 white spaces and also 1 space after //. so something like that:
__// filament_diameter>0.01 to enable  volumetric extrusion
instead of
//filament_diameter>0.01 to enable  volumetric extrusion

Reply:
ok